src/runtime/pkg/oci/utils.go

```diff
@@ -601,6 +609,29 @@ func addHypervisorHotColdPlugVfioOverrides(ocispec specs.Spec, sbConfig *vc.Sand
 	return nil
 }
 
+func addHypervisorPCIeRootPortOverrides(ocispec specs.Spec, sbConfig *vc.SandboxConfig) error {
```

Comment:
There is code duplication here. Can we consider condensing the two to a single method?

Reply:
Which two do you want to condense into a single method?

Reply:
The one is the switch-port and the other is the root-port ....